grizzly-ls/src/grizzly_ls/server/inventory.py

```diff
@@ -125,16 +125,23 @@ def compile_inventory(ls: GrizzlyLanguageServer, *, standalone: bool = False) ->
 
     try:
         ls.behave_steps.clear()
-        # only include paths that doesn't contain [unix] hidden directories
-        paths = [
-            path.parent
-            for path in ls.root_path.rglob('*.py')
-            if path.parent.is_dir()
-            and all(
-                exclude not in path.parent.as_posix()
-                for exclude in ['.', 'node_modules']
-            )
-        ]
+        # Ignore [unix] hidden files, node_modules and bin by default
+        file_ignore_patterns = (
+            ls.file_ignore_patterns
+            if ls.file_ignore_patterns
+            else ['**/.*', '**/node_modules', '**/bin']
+        )
+        paths = set(
+            [
+                path.parent
+                for path in ls.root_path.rglob('*.py')
+                if path.parent.is_dir()
+                and any(
```

Comment:
should it really be `any`?
doesn't that translate "include path if parent is a dir and at least one exclude pattern doesn't match", but it shouldn't match any of the ignore patterns?